server/resource_server/utils/utils.go

```diff
@@ -24,14 +24,18 @@ const SecretSize = 32
 
 var WorkingDirectory string
 
-// Response is used for static shape json return
 type Response struct {
 	Status  bool        `json:"status"`
 	Message string      `json:"message"`
-	Error   interface{} `json:"errors"`
 	Data    interface{} `json:"data"`
 }
 
+type ErrorResponse struct {
+	Status  bool        `json:"status"`
```

Comment:
Status should not be boolean, we need more clarity, maybe IsSuccess

Reply:
Okay, changed to IsSuccess.